Bigtable/src/ReadModifyWriteRowRules.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Bigtable\V2\ReadModifyWriteRule;
+
+/**
+ * Represents collection of ReadModifyWriteRule to perform operation on Bigtable table.
```

Comment:
`Represents a collection of read/modify/write rules specifying how the specified row's contents are to be transformed into writes. Entries are applied in order, meaning that earlier rules will affect the results of later ones. This is intended to be used in combination with {@see Google\Cloud\Bigtable\DataClient::readModifyWriteRow()}.`